src/components/Header.tsx

```diff
@@ -1,31 +1,164 @@
+										<a href='#' className='rounded-md bg-gray-900 px-3 py-2 text-sm font-medium text-white'>
+											Dashboard
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Team
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Projects
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Calendar
+										</a>
+									</div>
+								</div>
+							</div>
+							<div className='hidden sm:ml-6 sm:block'>
+								<div className='flex items-center'>
+									<Link href='#'>
+										<a className='text-white' target='_blank' rel='noopener noreferrer'>
+											Sign In
+										</a>
+									</Link>
+								</div>
+							</div>
+							<div className='-mr-2 flex sm:hidden'>
+								{/* Mobile menu button */}
+								<Disclosure.Button className='inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white'>
+									<span className='sr-only'>Open main menu</span>
+									{open ? (
+										<XIcon className='block h-6 w-6' aria-hidden='true' />
+									) : (
+										<MenuIcon className='block h-6 w-6' aria-hidden='true' />
+									)}
+								</Disclosure.Button>
+							</div>
+						</div>
+					</div>
+
+					<Disclosure.Panel className='sm:hidden'>
+						<div className='space-y-1 px-2 pt-2 pb-3'>
+							{/* Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" */}
+							<Disclosure.Button
+								as='a'
+								href='#'
+								className='block rounded-md bg-gray-900 px-3 py-2 text-base font-medium text-white'
+							>
+								Dashboard
+							</Disclosure.Button>
+							<Disclosure.Button
+								as='a'
+								href='#'
+								className='block rounded-md px-3 py-2 text-base font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+							>
+								Team
+							</Disclosure.Button>
+							<Disclosure.Button
+								as='a'
+								href='#'
+								className='block rounded-md px-3 py-2 text-base font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+							>
+								Projects
+							</Disclosure.Button>
+							<Disclosure.Button
+								as='a'
+								href='#'
+								className='block rounded-md px-3 py-2 text-base font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+							>
+								Calendar
+							</Disclosure.Button>
+						</div>
+						<div className='border-t border-gray-700 pt-4 pb-3'>
+							<div className='flex items-center px-5'>
+								<div className='flex-shrink-0'>
+									<img
```

Comment:
we don't have a profile image implemented yet.

can you use a username / email / part of an email instead?

Reply:
> we don't have a profile image implemented yet.
> 
> can you use a username / email / part of an email instead?

![no_profimage3](https://user-images.githubusercontent.com/67428719/187021798-18c13c17-c27e-4080-bff4-a6dfa789e1cb.png)


@ykdojo you mean removing the profile image and keeping the email as this?

Reply:
@ykdojo why not keep it there as a placeholder for now. I think we're going to implement it anyway.

Reply:
I'm ok either way!